transport-native-epoll/src/main/c/io_netty_channel_unix_Errors.c

```diff
@@ -24,22 +25,14 @@ static jclass channelExceptionClass = NULL;
 static jclass ioExceptionClass = NULL;
 static jclass closedChannelExceptionClass = NULL;
 static jmethodID closedChannelExceptionMethodId = NULL;
+static char* nettyClassName = NULL;
 
 /** Notice: every usage of exceptionMessage needs to release the allocated memory for the sequence of char */
 static char* exceptionMessage(char* msg, int error) {
-    if (error < 0) {
-        // some functions return negative values
-        // and it's hard to keep track of when to send -error and when not
-        // this will just take care when things are forgotten
-        // what would generate a proper error
-        error = error * -1;
-    }
-    //strerror is returning a constant, so no need to free anything coming from strerror
-    char* err = strerror(error);
-    char* result = malloc(strlen(msg) + strlen(err) + 1);
-    strcpy(result, msg);
-    strcat(result, err);
-    return result;
+    // strerror is returning a constant, so no need to free anything coming from strerror
+    // error may be negative because some functions return negative values. we should make sure it is always
+    // positive when passing to standard library functions.
+    return netty_unix_util_prepend(msg, strerror(error < 0 ? -error : error));
```

Comment:
@normanmaurer - Also note this change. Just found that is it doing the same thing as `netty_unix_util_prepend`